Lib/idlelib/editor.py

```diff
@@ -434,6 +435,11 @@ def set_line_and_column(self, event=None):
         line, column = self.text.index(INSERT).split('.')
         self.status_bar.set_label('column', 'Col: %s' % column)
         self.status_bar.set_label('line', 'Ln: %s' % line)
+        
+        selected = len(self.text.get(SEL_FIRST, SEL_LAST))
+        self.status_bar.set_label('selected',
+                                f"({selected} selected)" if selected else '',
+                                side=RIGHT)
```

Comment:
f-string is a bit overkill here; likely slower.
```suggestion
        chars = len(self.text.get(SEL_FIRST, SEL_LAST))
        self.status_bar.set_label('selected', '%d selected' % chars if chars else '')
```